### DISCUSSION TITLE
vite打包ant-design-vue错误提示

### DISCUSSION THREAD

**Discussion by 15643701048a** [vbenjs/vue-vben-admin#2136] [Q&A]
vben-admin 2.8版本，ant-design-vue 2.2.8版本打包yarn build 错误提示
[vite]: Rollup failed to resolve import "ant-design-vue/es/vc-table/src/ExpandableRow" from "src\views\message\group\model3.vue".
This is most likely unintended because it can break your application at runtime.
If you do want to externalize this module explicitly add it to
`build.rollupOptions.external`
加上
rollupOptions: {
   external: ['ant-design-vue'],
}
后打包成功，运行本地打包dist/index.html文件后，浏览器控制台显示
Uncaught TypeError: Failed to resolve module specifier "ant-design-vue". Relative references must start with either "/", "./", or "../".
有没有解决方法？

**Comment by i7eo** [comment]
1. 首先使用external时，不仅仅添加了 `external: ['ant-design-vue']` 就结束了，还得添加 `globals: { 'ant-design-vue':  ant-design-vue}` 才行，否则就会报你本地的这个错
2. 查看你的 `src\views\message\group\model3.vue` 文件，在引入 `ant-design-vue` 组件时，是否写了详细地址，例如：`ant-design-vue/es/vc-table/src/ExpandableRow` 我猜测这个可能是你的编辑器自动引入出了点问题。引入组件直接 `import { Table } from 'ant-design-vue'` 就行了
